README.md

```diff
@@ -181,9 +181,9 @@ Mondoo signs Microsoft Windows executables, PowerShell scripts, Linux packages a
 
 **Note:** The cmdlet `Get-AuthenticodeSignature` is currently not available in PowerShell Core, so you will need a Windows installation to run it.
 
-We use [Azure Trusted Signing](https://learn.microsoft.com/en-us/azure/developer/devcenter/azure-trusted-signing/overview), which issues short-lived code signing certificates that rotate approximately every 72 hours. As a result, the certificate thumbprint will change frequently and should not be used for validation.
+Since the `11.66.1` release, we have adopted [Azure Trusted Signing](https://learn.microsoft.com/en-us/azure/developer/devcenter/azure-trusted-signing/overview), which provides short-lived code signing certificates that rotate approximately every 72 hours. This means the certificate thumbprint will differ between releases and cannot be used as a reliable method of validation.
```

Comment:
The link https://learn.microsoft.com/en-us/azure/developer/devcenter/azure-trusted-signing/overview returns 404. Should https://learn.microsoft.com/en-us/azure/trusted-signing/overview be the updated link?